app/views/games/_table.html.erb

```diff
@@ -10,20 +10,19 @@
 <% else %>
   <table class="game-table table table-bordered table-striped">
     <tr>
-      <th>Name</th>
-      <% if show_console %>
-        <th>Console</th>
-      <% end %>
+      <th>Top Image</th>
+      <th width='80%'>Name</th>
     </tr>
     <% @matches.each do |game| %>
       <tr>
+        <td>
+          <%= game.pictures.empty? ? 'No image!' : link_to(image_tag(game.top_rated_picture.image_url, class: 'table-image'), game) %>
+        </td>
         <td class="game-table-name-cell">
-          <%= link_to game.name, game %>
           <%= link_to button_tag("Mark as duplicate", :class => "pull-right btn btn-mini btn-primary"), mark_as_duplicate_path(game), :method => :post, :class => "duplicate-link" %>
+          <h4><%= link_to game.name, game %></h4>
+          <%= link_to game.console.name, game.console if show_console %><p><%= game.pictures.count %> other images</p>
```

Comment:
Ah, here is the `show_console`. I wish github had a staging area for comments to avoid situations like this lol. So what does this look like when `show_console` is true?